### PR TITLE
Document validation changes to the container.ports.name field

### PR DESCRIPTION
**Pull request by derekwaynecarr**

### CODE REVIEW

File: whats_new/ose_3_0_release_notes.adoc [hunk 1]
```diff
@@ -534,6 +534,19 @@ or groups are also specified, they take priority over values set in
 use from user interfaces when your template is shown.
 * link:../dev_guide/builds.html#custom-secrets[Secrets can now be added to
 custom builds] and mounted at user-specified locations.
+* Validation of the `*container.ports.name*` field was changed:
+** If specified, the value must be less than 15 characters, have at least one letter [a-z]
```
Comment (ncdc) [inline whats_new/ose_3_0_release_notes.adoc]: No more than 15 / less than 16 if I'm reading https://github.com/openshift/origin/blob/6a99c8ccd1f92507ead04896cc9b5f38dafbc1a6/Godeps/_workspace/src/k8s.io/kubernetes/pkg/util/validation/validation.go#L108-L136 correctly

Comment (derekwaynecarr) [inline whats_new/ose_3_0_release_notes.adoc]: updated, thanks for the review.

**Comment by ncdc** [comment]
LGTM

**Comment by adellape** [comment]
:100:

**Pull request merged**